packages/admin-next/dashboard/src/routes/products/product-detail/components/product-variant-section/use-variant-table-columns.tsx

```diff
@@ -140,6 +140,50 @@ export const useProductVariantTableColumns = (
         },
       }),
       ...optionColumns,
+      columnHelper.accessor("inventory", {
+        header: () => (
+          <div className="flex h-full w-full items-center">
+            <span className="truncate">{t("fields.inventory")}</span>
+          </div>
+        ),
+        cell: ({ getValue }) => {
+          const inventory: InventoryItemDTO[] = getValue()
+
+          let availableCount = 0
+          const locations = {}
+
+          inventory.forEach((i) => {
+            i.location_levels.forEach((l) => {
+              availableCount += l.available_quantity
+              locations[l.id] = true
+            })
+          })
+
+          const locationCount = Object.keys(locations).length
+
+          const text = !availableCount
+            ? t("products.variant.tableItemNone")
+            : t("products.variant.tableItem", {
+                availableCount,
+                locationCount,
+                count: locationCount,
+              })
+
+          return (
+            <div className="flex h-full w-full items-center gap-2 overflow-hidden">
+              <Component style={{ marginTop: 1 }} />
```

Comment:
The component should only be there if it is part of a kit. See [this design section](https://www.figma.com/design/z3aUuOVWUKmdHH0ofmMpEV/Web-app-3.0?node-id=5247-194367&t=Tsum79X8DIyDYC34-4) for an overview of the different modes